python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -137,8 +146,15 @@ class GrapheneAssetNode(graphene.ObjectType):
         beforeTimestampMillis=graphene.String(),
         limit=graphene.Int(),
     )
+    assetObservations = graphene.Field(
```

Comment:
what's the use case for this resolver? just need to confirm that we need this resolver, since it exists on `GrapheneAsset`

Reply:
In the asset graph view, source assets now behave more like regular assets-- asset observations are sort of the asset materializations of source assets, they provide relevant metadata which we can e.g. display in sidebar